src/components/Login/Login.js

```diff
@@ -68,9 +68,20 @@ const isActivactionInProgress = (location) => {
   return parsedQuery && parsedQuery === 'true';
 };
 
+const message = '¡Hola! ¿Me ayudas con mi cuenta suspendida por falta de pago?'
```

Comment:
I know that it is hardcoded, but what do you thing about moving to the lenguage file anywary?

Or... add a TODO haha